.vscode/tasks.json

```diff
@@ -1,41 +1,44 @@
 {
   // See https://go.microsoft.com/fwlink/?LinkId=733558
   // for the documentation about the tasks.json format
-  "version": "0.1.0",
+  "version": "2.0.0",
   "tasks": [
     {
-      "taskName": "Install all dependencies",
+      "label": "Install all dependencies",
+      "group": "build",
       "command": "./node_modules/.bin/lerna",
-      "isShellCommand": true,
+      "type": "shell",
       "args": [
         "bootstrap"
       ]
     },
     {
-      "taskName": "Remove all dependencies",
+      "label": "Remove all dependencies",
```

Comment:
Since there is a task below to `Clean all generated files`, do we need this task? I personally just use `npm run clean` ... which does the same as task below.

Reply:
We have two different tasks:

 - clean files generated by our build (`dist`, `api-docs`, etc.) - this is `npm run clean`
 - remove `node_modules` in all packages - this is `npm run clean:lerna`